code/dataobjects/QueuedJobRule.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+use AsyncPHP\Doorman\Rule;
+
+/**
+ * @property Int $Processes
+ * @property Varchar $Handler
+ * @property Decimal $MinimumProcessorUsage
+ * @property Decimal $MaximumProcessorUsage
+ * @property Decimal $MinimumMemoryUsage
+ * @property Decimal $MaximumMemoryUsage
+ * @property Decimal $MinimumSiblingProcessorUsage
+ * @property Decimal $MaximumSiblingProcessorUsage
+ * @property Decimal $MinimumSiblingMemoryUsage
+ * @property Decimal $MaximumSiblingMemoryUsage
+ */
+class QueuedJobRule extends DataObject implements Rule {
+	/**
+	 * @var array
+	 */
+	private static $db = array(
+		'Processes' => 'Int',
+		'Handler' => 'Varchar',
+		'MinimumProcessorUsage' => 'Decimal',
+		'MaximumProcessorUsage' => 'Decimal',
+		'MinimumMemoryUsage' => 'Decimal',
+		'MaximumMemoryUsage' => 'Decimal',
+		'MinimumSiblingProcessorUsage' => 'Decimal',
+		'MaximumSiblingProcessorUsage' => 'Decimal',
+		'MinimumSiblingMemoryUsage' => 'Decimal',
+		'MaximumSiblingMemoryUsage' => 'Decimal',
+	);
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return int
+	 */
+	public function getProcesses() {
+		if($this->Processes) {
+			return $this->Processes->getValue();
+		}
+
+		return 1;
+	}
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return null|string
+	 */
+	public function getHandler() {
+		if($this->Handler) {
+			return $this->Handler->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @return null|float
+	 */
+	public function getMinimumProcessorUsage() {
+		if($this->MinimumProcessorUsage) {
+			return $this->MinimumProcessorUsage->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return null|float
+	 */
+	public function getMaximumProcessorUsage() {
+		if($this->MaximumProcessorUsage) {
+			return $this->MaximumProcessorUsage->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return null|float
+	 */
+	public function getMinimumMemoryUsage() {
+		if($this->MinimumMemoryUsage) {
+			return $this->MinimumMemoryUsage->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @return null|float
+	 */
+	public function getMaximumMemoryUsage() {
+		if($this->MaximumMemoryUsage) {
+			return $this->MaximumMemoryUsage->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return null|float
+	 */
+	public function getMinimumSiblingProcessorUsage() {
+		if($this->MinimumSiblingProcessorUsage) {
+			return $this->MinimumSiblingProcessorUsage->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return null|float
+	 */
+	public function getMaximumSiblingProcessorUsage() {
+		if($this->MaximumSiblingProcessorUsage) {
+			return $this->MaximumSiblingProcessorUsage->getValue();
+		}
+
+		return null;
+	}
+
+	/**
+	 * @inheritdoc
+	 *
+	 * @return null|float
+	 */
+	public function getMinimumSiblingMemoryUsage() {
+		if($this->MinimumSiblingMemoryUsage) {
```

Comment:
`return $this->getField('MinimumSiblingMemoryUsage')` would get the value directly.